takwimu/templates/takwimu/_includes/profile/sections_list.html

```diff
@@ -1,16 +1,41 @@
 {% load staticfiles %}
+<style>
+  .section-list-country-head {
+    text-align: center;
+    display: block;
+    font-size: 31px;
+    font-weight: bold;
+    line-height: 1.29;
+  }
+</style>
 
 <section class="py-5">
   <div class="container">
     {% if page.content_type.model == "profilesectionpage" %}
-      {% include 'takwimu/_includes/profile/country_selector.html' with current_page=page.get_parent %}
+      <div class="section-list-country-head">
+          <img class="rounded-circle border-light mr-2"
+            src={% with 'img/flags/'|add:page.get_parent.title|lower|add:'.svg' as image_static %}
+                  {% static image_static %}
+                {% endwith %}
+            border="8"
+            style="border-style:double; width: 68px; height: 67px;"/>
+          {{ page.get_parent.title }}
+      </div>
     {% else %}
-      {% include 'takwimu/_includes/profile/country_selector.html' with current_page=page %}
+      <div class="section-list-country-head">
+          <img class="rounded-circle border-light mr-2"
+            src={% with 'img/flags/'|add:page.title|lower|add:'.svg' as image_static %}
+                  {% static image_static %}
+                {% endwith %}
+            border="8"
+            style="border-style:double; width: 68px; height: 67px;"/>
+          {{ page.title }}
+      </div>
```

Comment:
Did you remove the country selection? I'd prefer that we translate edits to the re-usable component.

Reply:
From the design it looked like its not a button and has no drop-down selection. Shouldn't that be the case?